Library/Formula/subliminal.rb

```diff
@@ -15,93 +13,111 @@ class Subliminal < Formula
+    sha256 "a2b29bd048ca2fe54a046b29770964738872a9747003a371344a93eedf7ad58e"
   end
 
-  resource "dogpile.cache" do
-    url "https://pypi.python.org/packages/source/d/dogpile.cache/dogpile.cache-0.5.4.tar.gz"
-    sha1 "bc1dff9b2f1bbe2a09ca5e16d0d160a730ef8732"
+  resource "enzyme" do
+    url "https://pypi.python.org/packages/source/e/enzyme/enzyme-0.4.1.tar.gz"
+    sha256 "f2167fa97c24d1103a94d4bf4eb20f00ca76c38a37499821049253b2059c62bb"
+  end
+
+  resource "html5lib" do
+    url "https://pypi.python.org/packages/source/h/html5lib/html5lib-0.999.tar.gz"
+    sha256 "c3887f7e2875d7666107fa8bee761ff95b9391acdcc7cd1b5fd57a23b5fbc49e"
+  end
+
+  resource "stevedore" do
+    url "https://pypi.python.org/packages/source/s/stevedore/stevedore-1.3.0.tar.gz"
+    sha256 "beab2b7f91966d259796392c39ed6f260b32851861561dd9f3b9be2fd0c426a5"
   end
 
   resource "python-dateutil" do
-    url "https://pypi.python.org/packages/source/p/python-dateutil/python-dateutil-2.2.tar.gz"
-    sha1 "fbafcd19ea0082b3ecb17695b4cb46070181699f"
+    url "https://pypi.python.org/packages/source/p/python-dateutil/python-dateutil-2.4.1.tar.gz"
+    sha256 "23fd0a7c228d9c298c562245290a3f82999586c87aae71250f95f9894cb22c7c"
   end
 
-  resource "colorlog" do
-    url "https://pypi.python.org/packages/source/c/colorlog/colorlog-2.4.0.tar.gz"
-    sha1 "30b4a621ec372ba4ab367cb5446a4a4293f6e9ed"
+  resource "guessit" do
+    url "https://pypi.python.org/packages/source/g/guessit/guessit-0.9.4.tar.gz"
+    sha256 "88689713946faad4ae12467dff40a8f3daa629200328fbc5256093e96b1b0d19"
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", "#{libexec}/lib/python2.7/site-packages"
-    %w[six charade guessit pysrt html5lib enzyme chardet dogpile.core requests babelfish beautifulsoup4 dogpile.cache python-dateutil colorlog].each do |r|
-      resource(r).stage do
-        system "python", "setup.py", "install", "--prefix=#{libexec}", "--record=installed.txt"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
       end
     end
 
-    ENV.prepend_create_path "PYTHONPATH", "#{lib}/python2.7/site-packages"
-    system "python", "setup.py", "install", "--prefix=#{prefix}",
-           "--single-version-externally-managed", "--record=installed.txt"
-    bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    # dogpile is a namespace package and .pth files aren't read from our
+    # vendor site-packages
+    touch libexec/"vendor/lib/python2.7/site-packages/dogpile/__init__.py"
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
   test do
+    mkdir ".config"
```

Comment:
Might want to make this `#{ENV["HOME"]}/.config` just to clarify what's going on here?